redash/authentication/remote_user_auth.py

```diff
@@ -0,0 +1,34 @@
+import logging
+from flask import redirect, url_for, Blueprint, request
+from redash.authentication.google_oauth import create_and_login_user
+from redash.authentication.org_resolving import current_org
+from redash import settings
+
+logger = logging.getLogger('remote_user_auth')
+
+blueprint = Blueprint('remote_user_auth', __name__)
+
+@blueprint.route("/remote_user/login")
+def login():
+    next_path = request.args.get('next')
+
+    if not settings.REMOTE_USER_LOGIN_ENABLED:
+        logger.error("Cannot use remote user for login without being enabled in settings")
+        return redirect(url_for('index', next=next_path))
```

Comment:
Sorry, I overlooked it in previous review, but this should be `redash.index` now.